source/fundamentals.txt

```diff
@@ -25,20 +25,20 @@ Fundamentals
    /fundamentals/csfle
    /fundamentals/time-series
 
-The Fundamentals section contains detailed information to help you
-expand your knowledge on how to use MongoDB with the Java driver. We
-divide the information in this section into the following topics: 
+The Fundamentals section is where you will find coverage of
```

Comment:
[Reminder](https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/
): avoid "will" (future tense) and use present tense instead.